### PR TITLE
style: input fields same length

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

**Comment by RalfJung** [comment]
Not sure which input fields you mean, but alignment doesn't really seem to work here?

**Comment by rubo77** [comment]
The line break should be gone now.

 But what do you mean with the button size?

File: static/style.css [hunk 1]
```diff
@@ -9,7 +9,7 @@ body {
 }
 
 input[id=node]{
-    width: 222px;
+    width: 220px;
```
Comment (RalfJung) [inline static/style.css]: Pixel tweaking... that seems extremely fragile and dependent on the fonts the user has installed. Is there no better way?

**Comment by RalfJung** [comment]
> But what do you mean with the button size?

The two buttons are not aligned -- the one at the bottom is slightly more to the left, which is irritating. I thought aligning them was part of the point?

**Comment by rubo77** [comment]
I wanted to have the text and input field the same length as the selectbox.

It is fine now.

**Comment by RalfJung** [comment]
By "text and input field" you mean "Knoten ID manuell eingeben:   [input box]"?

They don't have the same length here even with your PR merged:
screenshot from 2018-06-19 13-18-01

The input box goes a little further to the right, and the two buttons are not aligned.

I think to make this look right, you need something box-based which actually tells the browser to align things.

**Comment by rubo77** [comment]
I think the input field has some margin or such, that differs on the resolution.

I think it would be solved if you wrap a div around the input field:

    <div id="inputwrap">
and style
  
    input[id=node]{
      width:100%;
    }
    inputwrap{
      display:inline-block;
      width:218px;
    }

(you can try a bit, I have to work another project today)

**Comment by RalfJung** [comment]
Fixed by fb1d2a92520a435038855877653bc559070e180e.

**Pull request closed without merging**